contributing/contributing_to_datacommons.md

```diff
@@ -21,18 +29,42 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
-## Code reviews
+### Becoming familiar with the project
+
+You can find suggested reading about the domain in which Data Commons operates in [the Background section](/contributing/background/background.html).
```

Comment:
I found "the domain in which Data Commons operates in" to be unclear and slightly intimidating. What about just "You can find suggested reading about Data Commons concepts in the Background section"?